### DISCUSSION TITLE
Schema Registry Client

### DISCUSSION THREAD

**Discussion by dkirrane** [gklijs/schema_registry_converter#102] [Q&A]
Can this be used like the Java [SchemaRegistryClient](https://github.com/confluentinc/schema-registry/blob/master/client/src/main/java/io/confluent/kafka/schemaregistry/client/SchemaRegistryClient.java)?

My main use-case is to validate a bunch of Avro schemas and then register them.
And another user-case to delete a Schema

**Comment by gklijs** [comment]
Not everything is there. For example, there are no compatibility checks. If you need specific functionality, please list those. It's hard to do everything the confluent schema registry clients can do. The main focus of this library is to be able to read and write in a way to be byte compatible with the Java ones. But I've opened up or added some things before.

It really helps to know specifically what would be missing for your use case. Deletion is also not in there but should be pretty easy to add, for example.

**Comment by gklijs** [comment]
I'm also still considering generating an (async) client based on the open API spec of the schema registry client. And then either use that as a dependency for this crate and/or just leave them separate.
